src/wiktextract/extractor/ru/models.py

```diff
@@ -24,6 +39,70 @@ class Sound(BaseModelWrap):
     )
 
 
+class Sense(BaseModelWrap):
+    raw_gloss: Optional[str] = Field(
+        default=None,
+        description="Raw gloss string for the word sense. This might contain tags and other markup.",
+    )
+    gloss: Optional[str] = Field(
+        default=None,
+        description="Gloss string for the word sense. This has been cleaned, and should be straightforward text with no tags.",
+    )
+    tags: list[str] = Field(
+        default=[],
+        description="List of tags affecting the word sense.",
+    )
+    notes: list[str] = Field(
+        default=[],
+        description="List of notes for the word sense. Usually describing usage.",
+    )
+    categories: list[str] = Field(
+        default=[],
+        description="list of sense-disambiguated category names extracted from (a subset) of the Category links on the page",
+    )
+    examples: list["Example"] = Field(
```

Comment:
Thanks! I think that's a mental leftover from using `typing.List` for so long.

Reply:
Interesting side note: When directly referring to the class, it has to be defined ahead of time. When using quotes, it can be defined later in the file.